.github/workflows/push_trigger.yml

```diff
@@ -7,6 +7,7 @@ on:
       - master
       - 1.1.1
       - develop
+      - 1.1.2
```

Comment:
please add like this -

    branches:
      - master
      - 1.*
      - develop